src/medium-editor-md.js

```diff
@@ -26,10 +26,12 @@ module.exports = function (options, callback) {
     options.events = options.events || ["input", "change"];
     callback = callback || options.callback || function () {};
 
-    // Wait until the Medium Editor is inited
-    setTimeout(function() {
+    // Called by medium-editor during init
+    this.init = function(mediumEditorInstance) {
```

Comment:
You can make how many changes ~~you want~~ are needed. :smile: Git & GitHub do a nice thing for us showing the diffs.

If `this.me.elements` contains the DOM elements of the extension where it was loaded, that's awesome! :sparkles: Then the `element` option is obsolete (I added it because I didn't know how to get the ME instance).

What happens if there is more than one editor?